### DISCUSSION TITLE
Welcome to lwt Discussions!

### DISCUSSION THREAD

**Discussion by HugoFara** [HugoFara/lwt#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rickvw0604** [comment]
Hi, I am trying to use the app on a webhoster. When I want to add a text I get the following error:

![image](https://user-images.githubusercontent.com/115886024/207852431-4be29ad1-ff62-4ba9-8828-7831be12a370.png)

Do you know what to do?

**Comment by HugoFara** [comment]
Hi Rick! It is a know bug, I am trying to get rid of it with the next version of LWT. You nay have seen #20 and #40, here is the general solution. As you are using a web hoster, you can refer to the last paragraph.

## Speed over security
Allow local infiles in MySQL and PHP, ``mysqli.allow_local_infile = 1``  (in ``php.ini``) tells PHP that it can is LOCAL INFILE, but you need to set the permission in MySQL as well. Run ``mysql``, and then ``SET GLOBAL local_infile=1;``. It is a good option on local servers (for your computer), but can lead to security issues on web servers.
## Secure way
just disable everything that looks like mysqli.allow_local_infile in ``php.ini``, as I have duplicated code to do things in pure PHP.

For web hosters, the idea stays the same but instead of modifying the files you should probably go in the options and find the checkbox speaking about enabling "local infiles".

The last solution is to edit LWT files directly. Replace ``get_first_value("SELECT @@GLOBAL.local_infile as value")`` by ``false`` in ``inc/database-connect.php`` (it appears two times). It will force things to work.

**Reply by Rickvw0604** [reply]
Hi Hugo thx for your help! Sadly I encountered another issue:
![image](https://user-images.githubusercontent.com/115886024/207860917-3bcb3abe-8d3c-4e6a-89ee-3802724f7dcb.png)

Is this a known bug too? Thank you!

**Reply by HugoFara** [reply]
Yes, you are using a deprecated version of PHP, you nay try to update to PHP >=8 as PHP 7 is reaching end-of-support in 20 days...

**Reply by Rickvw0604** [reply]
Now this pops up:
![image](https://user-images.githubusercontent.com/115886024/207864874-fb1504a5-cf85-42fd-a6ef-e3d1ca4975c9.png)

this is the code in my file:
![image](https://user-images.githubusercontent.com/115886024/207864973-5e6c7151-f4ab-4bd4-a41d-823438010c8c.png)
